packaging/macOS/README.md

```diff
@@ -0,0 +1,90 @@
+MacOS installer for Parsec
+==========================
+
+Build steps
+-----------
+
+### 1 - Use a compatible Python version
+
+The app building is done with PyInstaller. For it to work properly, you must use a version of Python anterior to 3.8, otherwise the app won't launch at all. Any 3.6.x or 3.7.x version should work fine.
+
+
+### 2 - Build the application
+
+To make the .app:
+
+```shell
+sh macos_pyinstaller.sh
+```
+
+If a version is already built, you can use the `-f` or `--force` option to bypass the confirmation dialogs when building the new one.
+You can also use the `-i` or `--install` option to directly install the application in the local `/Applications` folder after building.
+
+
+### 3 - Sign the application
+
+```shell
+codesign --deep -s <identity> --timestamp -v --entitlements entitlements.plist -o runtime dist/parsec.app
+```
+
+`identity` being the signing certificate, to be downloaded with proper access here : https://developer.apple.com/account/resources/certificates/list
+
+You can check the codesign using:
+```shell
+codesign -vvv --deep --strict dist/parsec.app
+```
+The next step can only be successful if this check is.
+
+
+### 4 - Zip and Notarize the app
+
+The Apple ID to be used is usually the email address used for the Apple account, and an application-specific password for this script has to be generated here : https://appleid.apple.com/account/manage
+
+It can then be stored in the keychain:
+```shell
+xcrun altool --store-password-in-keychain-item "AC_PASSWORD" -u "your-apple-id" -p "your-password"
+```
+
+The app now has to be .zipped to then be notarized using the ID and password:
```

Comment:
`has to be zipped` without the `.` (I thought Apple created a brand new `.zipped` format ^^)